### ISSUE TITLE
bias present in pretrained network

### ISSUE THREAD

**Issue by jonathanbonnard** [d-li14/mobilenetv2.pytorch#11]
Hi, 
I don't understand why there are biases in your models. In model definition, conv2d has bias set to false :
nn.Conv2d(hidden_dim, oup, 1, 1, 0, bias=False) 
Is this normal?

**Comment by d-li14** [comment]
The values of bias are negligible compared with the weight, so they can be approximated to zero.

**Comment by jonathanbonnard** [comment]
It turns out that  the biases are the gamma parameters of the batch normalization layers.
I am sorry. Thank you anyway d-li14. Have a nice day
